examples/aio-sx/app-armor/app-armor.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: starlingx.windriver.com/v1
```

Comment:
We normally require a patch applied to a specific type of system for example. You'll need to create a patch based on aio-sx.yaml, and when you issue a make, the patch will be applied to create a new configuration by kusomize tool

Reply:
Added the required configuration